github-sync/github-data/repositories.yaml

```diff
@@ -1137,6 +1139,10 @@ repositories:
         requiredApprovingReviewCount: 1
         statusChecks:
           - DCO
+          - CI Test / yamllint
+          - CI Test / test
+          - CI Test / lint
+          - Cosign tests / validate
```

Comment:
Do we not want "TUF Client tests / client" here?

Reply:
YES! That was the one I was peeved by in particular :P